sdk/src/main/java/com/mapbox/maps/MapView.kt

```diff
@@ -296,9 +300,65 @@ open class MapView : FrameLayout, MapPluginProviderDelegate, MapControllable {
    */
   @SuppressLint("ClickableViewAccessibility")
   override fun onTouchEvent(event: MotionEvent): Boolean {
+    interceptedActionList.firstOrNull()?.let {
```

Comment:
Shouldn't all events be passed and not just first one?

Reply:
E.g., instead of the current setup - 
```
if (intereceptedEvents.notEmpty) {
  intereceptedEvents.forEach { 
    mapController.onTouchEvent(it)
  }
  interceptedActionList.clear()
}
```

Reply:
done